cypress/e2e/pages/testsForAllPages.js

```diff
@@ -28,6 +32,23 @@ export const testsThatAlwaysRunForAllPages = ({
       }
     });
 
+    it('should have href values correctly formed', () => {
+      const allowedUrls = ['programmes/p0703hz7', 'send/u94753086'];
+
+      cy.get('a[href^="https://www.bbc.com"]').each($tag => {
```

Comment:
Let's only check the links within main?
```suggestion
      cy.get('main a[href^="https://www.bbc.com"]').each($tag => {
```

or if that doesn't work
```suggestion
      cy.get('main > a[href^="https://www.bbc.com"]').each($tag => {
```

And then we could probably remove the footer check